packages/logger/src/index.ts

```diff
@@ -93,6 +96,100 @@ function createDisabledLogger (namespace: string): debug.Debugger {
   return logger
 }
 
+export interface ComponentLoggerOptions {
+  prefixLength: number
+  suffixLength: number
+}
+
+/**
+ * Create a component logger that will prefix any log messages with a truncated
+ * peer id.
+ *
+ * @example
+ *
+ * ```js
```

Comment:
we really should have these examples as TS and then add the capability for aegir to lint them.

Reply:
They're marked as ts now 👍